src/literals.rs

```diff
@@ -15,8 +15,13 @@ use memchr::{memchr, memchr2, memchr3};
 use syntax;
 
 use freqs::BYTE_FREQUENCIES;
+
+#[cfg(feature = "simd-accel")]
```

Comment:
You still import it as `simd_fallback`, though. Might be a good idea to provide a shim module in `lib.rs` depending on the feature flag (the shim in `simd_accel` only deals with the target arch feature).